megatron/core/distributed/fsdp/src/megatron_fsdp/param_and_grad_buffer.py

```diff
@@ -94,8 +101,8 @@ def _p_assert(cond: Any, s: str, raise_assertion_error: bool = True) -> None:
     message ``s`` since otherwise, it is swallowed.
     """
     if not cond:
-        print(s)
-        traceback.print_stack()
+        logger.error(s)
+        logger.error(''.join(traceback.format_stack()))
```

Comment:
```suggestion
        logger.error(s, exc_info=True)
```

Reply:
Hi @Skylion007, thanks for the comments! Glad to have such an active user like you. This PR is actually a cherry-pick for dev branch. Its change is already merged into main branch here: https://github.com/NVIDIA/Megatron-LM/commit/268fda08592528b7bc1a21aadaed259980ca8efb

Reply:
Sigh, the graph I posted was from that commit then. :(

Reply:
@shjwudp @xuwchen Are you aware of this issue?